systemtest/src/test/java/io/strimzi/systemtest/SecurityST.java

```diff
@@ -744,6 +749,130 @@ void testNetworkPoliciesWithTlsListener() throws Exception {
         assertThrows(AssertionError.class, () -> availabilityTest(50, CLUSTER_NAME, true, topic1, kafkaUser, kafkaClientsNewPodName));
     }
 
+    @Test
+    void testTlsHostnameVerificationWithKafkaConnect() {
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 3, 1).done();
+        LOGGER.info("Getting IP of the bootstrap service");
+
+        String ipOfBootstrapService = kubeClient().getService(KafkaResources.bootstrapServiceName(CLUSTER_NAME)).getSpec().getClusterIP();
+
+        LOGGER.info("Kafka connect without config {} will not connect to {}:9093", "ssl.endpoint.identification.algorithm", ipOfBootstrapService);
+
+        KafkaConnect kafkaConnect = testMethodResources().kafkaConnectWithoutWait(testMethodResources().defaultKafkaConnect(CLUSTER_NAME, 1)
+                .editMetadata()
+                    .addToLabels("type", "kafka-connect")
+                .endMetadata()
+                .editSpec()
+                    .withNewTls()
+                        .addNewTrustedCertificate()
+                            .withSecretName(CLUSTER_NAME + "-cluster-ca-cert")
+                            .withCertificate("ca.crt")
+                        .endTrustedCertificate()
+                    .endTls()
+                    .withBootstrapServers(ipOfBootstrapService + ":9093")
+                .endSpec()
+                .build());
+
+        StUtils.waitUntilPodIsPresent(CLUSTER_NAME + "-connect");
+
+        String kafkaConnectPodName = kubeClient().listPods("type", "kafka-connect").get(0).getMetadata().getName();
+
+        StUtils.waitUntilPodIsInCrashLoopBackOff(kafkaConnectPodName);
+
+        assertThat("CrashLoopBackOff" ,is(kubeClient().getPod(kafkaConnectPodName).getStatus().getContainerStatuses().get(0)
```

Comment:
is that assert rly needed? wait on line 780 do the same thing

Reply:
Yes it is, because firstly the pod will be in the status as follows:
```
1) ContainerCreating
2) Runnning
3) Error
4) Running 
5) Error
6) CrashLoopBackOff
```
I know it's another wait but overall it takes like 10s and from my POV it's more semantically better.